### PR TITLE
zh-trans: /docs/tasks/administer-federation/events.md

### PR DESCRIPTION
**Pull request by zgfh**
zh-trans: /docs/tasks/administer-federation/events.md

### CODE REVIEW

File: content/zh/docs/tasks/administer-federation/events.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+<!--
+This guide assumes that you have a running Kubernetes Cluster
+Federation installation. If not, then head over to the
+[federation admin guide](/docs/concepts/cluster-administration/federation/) to learn how to
+bring up a cluster federation (or have your cluster administrator do
+this for you). Other tutorials, for example
+[this one](https://github.com/kelseyhightower/kubernetes-cluster-federation)
+by Kelsey Hightower, are also available to help you.
+-->
+本指南假设您安装好了 Kubernetes 集群联邦。如果没有，请先查看 [federation admin guide](/docs/admin/federation/) 来学习怎样安装集群联邦（也可以让您的管理员替您完成安装）。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-federation/events.md]:
```suggestion
本指南假设您已安装好了 Kubernetes 集群联邦。如果没有，请先查看 [联邦管理指南](/docs/admin/federation/) 来学习怎样安装集群联邦（也可以让您的管理员替您完成安装）。
```

File: content/zh/docs/tasks/administer-federation/events.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+其他的教程，例如 Kelsey Hightower 编写的 [这个教程](https://github.com/kelseyhightower/kubernetes-cluster-federation) 也会帮到你。
+
+<!--
+You are also expected to have a basic
+[working knowledge of Kubernetes](/docs/setup/) in
+general.
+-->
+您还应该具备 Kubernetes 的基本工作知识。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-federation/events.md]:
```suggestion
您还应该具备 [Kubernetes 的基本操作](/docs/setup) 的知识。
```

File: content/zh/docs/tasks/administer-federation/events.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+-->
+## 查看联邦事件
+
+<!--
+Events in federation control plane (referred to as "federation events" in
+this guide) are very similar to the traditional Kubernetes
+Events providing the same functionality.
+Federation Events are stored only in federation control plane and are not passed on to the underlying Kubernetes clusters.
+-->
+联邦控制平面的事件（本指南中称为“联邦事件”）与提供相同功能的传统 Kubernetes 事件非常相似。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-federation/events.md]:
```suggestion
联邦控制平面的事件（本指南中称为“联邦事件”）与传统的 Kubernetes 事件非常相似，提供相同功能。
```

File: content/zh/docs/tasks/administer-federation/events.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+联邦事件仅存储在联邦控制平面中，不会传递给基础 Kubernetes 集群。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-federation/events.md]:
```suggestion
联邦事件仅存储在联邦控制平面中，不会传递给下层 Kubernetes 集群。
```

File: content/zh/docs/tasks/administer-federation/events.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+您可以通过运行以下命令从联邦 API 服务器获取所有事件：
+
+```shell
+kubectl --context=federation-cluster get events
+```
+
+<!--
+The standard kubectl get, update, delete commands will all work.
+-->
+标准的 kubectl get，update，delete 命令都可以正常工作。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-federation/events.md]:
```suggestion
标准的 kubectl get、update、delete 命令都可以正常工作。
```

**Comment by zgfh** [comment]
@tengqm 感谢，已经修改

**Pull request merged**